### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database Properties Preview_Filegroup_Screen Reader: Screen reader is not announcing about the "Checkbox" in "Rows" table under "Default" and "Autogrow All Files" column header in the "Filegroup" tab.

### ISSUE THREAD

**Issue by Raj-ghuge** [microsoft/azuredatastudio#25210]
### Actual results:
Screen reader is not announcing about the "Checkbox" in "Rows" table under "Default" and "Autogrow All Files" column header in the "Filegroup" tab.

**Note: Issue seen throughout the application for similar type control.**

### Expected results:
Screen reader should announce about the "Checkbox" in "Rows" table under "Default" and "Autogrow All Files" column header in the "Filegroup" tab.

### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG1.3.1; #Info and Relationships; #FTP; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio; #Linux; #Element:Checkbox; #ScreenReader; #Orca; 

### Environment Details:
Application Name: Azure Data Studio
OS Name: Ubuntu 22.04.2 LTS
Azure data studio 1.47.0
Screen Reader: ORCA

### Repro Steps:

1.  Turn on ORCA.
2.  Install Azure data studio application and open it
3.  Make sure to enable "Preview" feature in settings.
4.  Navigate by using TAB key till "Database" and press enter to expand the tree.
5.  Navigate by using TAB key till any database and pressF10 to open context menu.
6.  Navigate by using TAB key till "Properties Preview" and press enter.
7.  Navigate by using TAB key till "Filegroup" tab and press enter.
8.  Navigate by using TAB key till "Database files" table.
9.  Navigate by using arrow key till "Checkbox."
10.  Observe that, screen reader is not announcing about the "Checkbox."

### User Impact:
Screen Reader user will be facing difficulties to understand checkbox is present in the "Rows" table.

### Attachments:
[25210.zip](https://github.com/microsoft/azuredatastudio/files/13948855/25210.zip)

**"Please do not close this bug. This bug should only be closed by Trusted Tester after verification."**

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by corivera** [comment]
@Raj-ghuge This issue has been fixed in the latest Ubuntu Insiders version of ADS here: https://azuredatastudio-update.azurewebsites.net/latest/linux-deb-x64/insider

Can you verify that the issue is resolved?

**Comment by Raj-ghuge** [comment]
Verified the issue in the environment- 
Azure data studio 1.49.0 insider
The issue is fixed hence closing the bug. PFA.
#Regressed:03-08-24; #Closed

https://github.com/microsoft/azuredatastudio/assets/130441232/90167bbf-2776-4a21-aafe-48cacb7aac6a
